pkgs/zoltan.yaml

```diff
@@ -0,0 +1,32 @@
+extends: [autotools_package]
+dependencies:
+  build: [mpi, {{build_with}}]
+
+sources:
+- key: tar.gz:lpoumvepxhdtwis3xtz5ebwfldbrrszj
+  url: http://www.cs.sandia.gov/~kddevin/Zoltan_Distributions/zoltan_distrib_v3.8.tar.gz
+
+defaults:
+  #include/Makefile.export.zoltan.macros
+  relocatable: false
+
+build_stages:
+
+  - name: setup_builddir
+    after: prologue
+    handler: bash
+    bash: |
+      mkdir -p _build
+      cd _build
+
+  - name: setEnv
+    after: setup_builddir
+    handler: bash
+    bash: |
+      export MPI_DIR=$(dirname $(dirname $MPICC) )
+      export LD_LIBRARY_PATH=${MPI_DIR}/lib
```

Comment:
This is very hackish to set `LD_LIBRARY_PATH`, even if during the build only. Usually there is a better solution. But that's fine for now to get something that works.